webapp/public/styles.css

```
@@ -0,0 +1,1063 @@
+/* ================================================================
+   Career Ops Dashboard — Claude Design System
+   Based on DESIGN.md: warm parchment, terracotta, editorial type
+   ================================================================ */
+
+/* ─── Google Fonts ─────────────────────────────────────────────── */
+@import url('https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;1,400&family=Inter:wght@400;500;600&family=JetBrains+Mono:wght@400;500&display=swap');
```

Stylelint (17.6.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;1,400&family=Inter:wght@400;500;600&family=JetBrains+Mono:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;1,400&family=Inter:wght@400;500;600&family=JetBrains+Mono:wght@400;500&display=swap'" (import-notation)

(import-notation)